source/fundamentals/configure.txt

```diff
@@ -0,0 +1,101 @@
+.. _entity-framework-configure:
+
+===========================================
+Configure Entity Framework Core for MongoDB
+===========================================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: EF, EF Core, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+In this guide, you will learn how to configure an application to use the
+{+provider-long+}. To learn how to set up a new project and install the
+{+provider-short+}, see the :ref:`entity-framework-quickstart`.
+
+Create a POCO
+-------------
+
+Create a `Plain old CLR/Class object
+<https://en.wikipedia.org/wiki/Plain_old_CLR_object>`__, or **POCO**, to use as
+a model for your entity. A POCO is a simple class object that doesn't inherit
+features from any framework-specific base classes or interfaces.
+
+To learn more about POCOs, see the :driver:`POCO guide
+</csharp/current/fundamentals/serialization/poco/>` in the
+{+csharp-driver-short+} documentation.
+
+The following code example shows how to create a Customer POCO:
+
+.. literalinclude:: /includes/fundamentals/code-examples/configure/Customer.cs
+   :language: csharp
+   :copyable:
+
+Create a DB Context Class
+-------------------------
+
+To begin using Entity Framework, create a class that derives from 
+`DBContext <https://learn.microsoft.com/en-us/dotnet/api/system.data.entity.dbcontext>`__.
+The ``DBContext`` class represents a session with the database and is used to
```

Comment:
S: The wording "session with the database" sounds a little foreign. "Database session" is probably more common

```suggestion
The ``DBContext`` class represents a database session and is used to
```